src/assets/scripts/client/InputController.js

```diff
@@ -41,15 +41,15 @@ export default class InputController {
      * @param aircraftCommander {AircraftCommander}
      * @param scopeModel {ScopeModel}
      */
-    constructor($element, aircraftCommander, aircraftController, scopeModel) {
+    constructor($element, aircraftController, scopeModel) {
         this.$element = $element;
         this.$body = null;
         this.$window = null;
         this.$commandInput = null;
         this.$canvases = null;
 
         this._eventBus = EventBus;
-        this._aircraftCommander = aircraftCommander;
+        this._aircraftCommander = aircraftController.getAircraftCommander();
```

Comment:
I don't think we need this.  we could access the commander via `this._aircraftController.aircraftCommander.runCommands()`, provided we move that to a getter (see below)

Reply:
done